example/controller-registration.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   deployment:
```

Comment:
Changing an internal chart should not lead to a different controller-registration.yaml, this file is probably wrongly generated.

Reply:
reverted